### ISSUE TITLE
[Feature request] Support qtwebview on Windows

### ISSUE THREAD

**Issue by quantonganh** [therecipe/qt#463]
We have a project that uses `qtwebview` but as you said [here|https://github.com/therecipe/qt/issues/455#issuecomment-330643831]:

> the WebView and the WebEngine are not supported on windows atm

Do you have a plan to support it?

**Comment by therecipe** [comment]
@quantonganh 
Yes maybe with Go 1.11, once these issue are resolved: https://github.com/golang/go/issues/17014 and https://github.com/golang/go/issues/20982

**Comment by petergloor** [comment]
Go 1.11 is out. Any updates here?

**Comment by therecipe** [comment]
@petergloor There was some promising activity recently: https://go-review.googlesource.com/c/go/+/133946
I will look into this.

**Comment by zanitee** [comment]
@therecipe windows not supporting the webview is one of the only major missing features I’ve yet to stumble across since using this binding, so well done thus far but I hope you can support windows soon.

**Comment by easykoo** [comment]
any update on this?

**Comment by therecipe** [comment]
The WebEngine/WebView modules should work on windows now.

To get this working you will need to install the 64 bit MSVC 2017 built of Qt and build yourself a custom Go version with these patches https://go-review.googlesource.com/c/go/+/133946
Instructions how you can do this can be found [here](https://github.com/therecipe/qt/wiki/Installation-on-Windows#official-version-with-webenginewebview-support) now

And there is also a new docker image available called `windows_64_shared_msvc` which can be used as well.
